src/loading-screen.css.ts

```diff
@@ -0,0 +1,99 @@
+import {keyframes, style, styleVariants} from '@vanilla-extract/css';
+import {sprinkles} from './sprinkles.css';
+import {vars as skinVars} from './skins/skin-contract.css';
+
+export const loadingScreen = style([
+    sprinkles({
+        display: 'flex',
+        flexDirection: 'column',
+        alignItems: 'center',
+    }),
+    {
+        minHeight: '100vh',
+    },
+]);
+
+export const inOutAnimationMs = 500;
+const backgroundAnimationTiming = `${inOutAnimationMs}ms cubic-bezier(.1,0,.7,1)`;
+const backgroundFadeIn = keyframes({
+    '0%': {
+        opacity: 0,
+    },
+    '100%': {
+        opacity: 1,
+    },
+});
+
+export const screenBackgroundAnimated = style({
+    animation: `${backgroundFadeIn} ${backgroundAnimationTiming}`,
+});
+
+export const screenBackground = styleVariants({
+    default: {
+        background: skinVars.colors.background,
+    },
+    inverse: {
+        background: skinVars.colors.backgroundBrand,
+    },
+});
+
+export const screenBackgroundFadeOut = style({
+    transition: `opacity ${backgroundAnimationTiming}`,
+    opacity: 0,
+});
+
+export const loadingScreenChildren = sprinkles({
+    flex: 1,
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+    justifyContent: 'center',
+});
+
+export const loadingScreenText = sprinkles({
+    paddingTop: 24,
+});
+
+const textAnimationTiming = `${inOutAnimationMs}ms cubic-bezier(.1,0,.7,1)`;
```

Comment:
```suggestion
const textAnimationTiming = `${inOutAnimationMs}ms cubic-bezier(0.215, 0.61, 0.355, 1)`;
```

works better with text animations (same as feedbacks Screen)

1000ms for this animation works better.. is it important to use the same inOutAnimationMs here?